tests/integration/test_design.py

```diff
@@ -2669,3 +2670,35 @@ def check_list_equality(lines, expected_lines):
         "    |---(body) nested_1_nested_1_comp_1_circle",
     ]
     assert check_list_equality(lines, ref) is True
+
+
+def test_design_parameters(modeler: Modeler):
+    """Test the design parameters functionality."""
+
```

Comment:
Seems like the test is failing - do you know the reason? There seems to be some kind of issue with the server apparently... We are using the latest proto and servers to check this so... maybe a pending PR on the server side?